Makefile

```diff
@@ -372,7 +372,14 @@ endif
 
 check-units:
 
-check: check-units installcheck pytest
+check: check-units installcheck pytest check-protos
+
+check-protos: $(ALL_PROGRAMS)
+ifeq ($(PYTEST),)
+	@echo "py.test is required to run the protocol tests, please install using 'pip3 install -r requirements.txt', and rerun 'configure'."
```

Comment:
should this do `false` at the end?

Reply:
I don't think I quite get your meaning here.

Reply:
Ah I see, I missed the `exit 1` from where i copy-pasta'd this from :facepalm: